src/LndClient/RPC/Silent.hs

```diff
@@ -99,3 +104,32 @@ ensureHodlInvoice env req = do
             AddInvoice.paymentRequest = Invoice.paymentRequest x,
             AddInvoice.addIndex = Invoice.addIndex x
           }
+
+closeChannelSync ::
+  (MonadUnliftIO m) =>
+  LndEnv ->
+  CloseChannelRequest ->
+  m (Either LndError ())
+closeChannelSync env req = do
+  cs0 <- listChannels env (ListChannels.ListChannelsRequest False False False False Nothing)
+  case cs0 of
+    Left err -> pure $ Left err
+    Right x ->
+      case filter (\ch -> channelPoint req == Channel.channelPoint ch) x of
+        [] -> return $ Left $ LndError "Cannot close channel that is not active"
```

Comment:
It just means it's already closed or pending, I think should return `Right ()`

Reply:
Agree, done